mcs/class/System/System.Diagnostics/PerformanceCounterCategory.cs

```diff
@@ -40,31 +40,41 @@ public sealed class PerformanceCounterCategory
 		private string machineName;
 		private PerformanceCounterCategoryType type = PerformanceCounterCategoryType.Unknown;
 
+// FIXME Span<char>?
+
 		[MethodImplAttribute (MethodImplOptions.InternalCall)]
-		static extern bool CategoryDelete (string name);
+		unsafe static extern bool CategoryDelete (char* category, int category_length);
```

Comment:
What's the need/benefit of replacing string with char+length?

Reply:
That's for the coop handles work, it's so we don't pass a `MonoObject*` to managed, but a `char*`.